### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Available Software_ Link Purpose: “More” link is not descriptive enough for the user to understand.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3358]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier
 
GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.4.4; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
 
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** MAS2.4.4

 **NOTE: Same issue is observed in “Install Details” screen as well.**

**Repro Steps:** 
1) Start Screen Reader and 0pen Java EE - Eclipse application.
2) Tab till Help-->Install New software.
3)  Enter http://dl.microsoft.com/eclipse/ in 'Work with' text box and press enter.
4)Verify “More” link is descriptive or not.

**Actual Result:** 
“More” link is not descriptive enough for the user to understand.

 **Expected Result:** 
“More” link should be descriptive enough for the user to understand.

**User Impact:** 
Users may not understand where that link will navigate to if links are not descriptive.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}
![Bug_3358_Link Purpose](https://user-images.githubusercontent.com/38523250/62208308-0f148900-b3b4-11e9-88de-880b5712aba2.png)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The install UI is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here is a screen shot of install AWS Eclipse Toolkits, we could see the same UI as this issue.

![image](https://user-images.githubusercontent.com/12445236/64940536-c9921980-d896-11e9-9997-4de9b5e87512.png)

**Comment by Flanker32** [comment]
- Importance of experience: Little, it's not a common behavior to click the more link while installing Eclipse Toolkits, and the more information didn't block user install our toolkits.
- Quality of experience: Little, users could click this link and they will know the information by the following UI.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749792
